indico/modules/auth/controllers.py

```diff
@@ -472,6 +474,11 @@ def check_email_unique(self, email, **kwargs):
                 if User.query.filter(~User.is_deleted, ~User.is_pending, User.all_emails == email).has_rows():
                     raise ValidationError('Email already in use')
 
+            @validates('accept_registration_terms')
+            def check_registration_terms(self, value):
+                if not value:
+                    raise ValidationError('Terms must be accepted')
```

Comment:
You could also pass `validate=validate_with_message(bool, 'Terms must be accepted')` in the field constructor above. Or even use `validate=not_empty` since the user should never see the error message anyway ;)

Reply:
Nice, using validate=not_empty now.